sos/report/plugins/openstack_barbican.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2019 Mirantis, Inc., Denis Egorenko <degorenko@mirantis.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenStackBarbican(Plugin, DebianPlugin, UbuntuPlugin):
+    """OpenStack Barbican Secure storage service"""
+    plugin_name = "openstack_barbican"
+    profiles = ('openstack', 'openstack_controller')
+
+    packages = (
+        'barbican-common',
+        'barbican-keystone-listener',
+        'barbican-worker'
+    )
+
+    requires_root = False
+
+    def setup(self):
+        self.add_copy_spec("/etc/barbican/")
+
+        if self.get_option("all_logs"):
+            self.add_copy_spec("/var/log/barbican/*")
+        else:
+            self.add_copy_spec("/var/log/barbican/*.log")
+
+        self.add_forbidden_path("/etc/barbican/*.pem")
+        self.add_forbidden_path("/etc/barbican/alias/*")
+
+    def postproc(self):
+        self.do_file_sub(
+            "/etc/barbican/barbican.conf",
+            r"(password[\t\ ]*=[\t\ ]*)(.+)",
+            r"\1********"
+        )
+
+        self.do_file_sub(
+            "/etc/barbican/barbican.conf",
+            r"(rabbit_password[\t\ ]*=[\t\ ]*)(.+)",
```

Comment:
Isn't this regexp already included in the previous one?

If not included, does not it make sense to merge the two REs into one, e.g.:

    `r"([password|rabbit_password][\t\ ]*=[\t\ ]*)(.+)"`

?

Reply:
Hi @pmoravec, so I've merged those into one, verified the regex works and resubmitted.